chart/otomi/templates/job.yaml

```diff
@@ -17,10 +17,8 @@ spec:
         runAsGroup: 999
       initContainers:
         - name: bootstrap-values
-          image: {{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}
-          {{- with .Values.image.pullPolicy }}
-          imagePullPolicy: {{ . }}
-          {{- end }}
+          image: otomi/core:{{ .Values.otomi.version | default .Chart.AppVersion }}
```

Comment:
`.Chart.AppVersion` refers to `appVersion: 'APP_VERSION_PLACEHOLDER'`. Is this a valid `otomi/core` tag?

Reply:
This is a placeholder. Then it will be substituted with the otomi version by github workflow